internal/daemon/controller/handlers/targets/registry.go

```diff
@@ -39,7 +39,7 @@ type registryEntry struct {
 }
 
 type registry struct {
-	sync.Map
+	*sync.Map
```

Comment:
😱 was this buggy the whole time?

> A Map must not be copied after first use

https://pkg.go.dev/sync#Map

Reply:
Yep! It was!